.github/workflows/release.yaml

```
@@ -42,6 +42,9 @@ jobs:
         with:
           go-version: '${{ env.GO_VERSION }}'
 
+      - name: check out release tag
```

Nit: "check out" means to view
https://promova.com/confusing-words/checkout-check-out

```suggestion
      - name: checkout release tag
```